### ISSUE TITLE
Add sub-directories to upload folder

### ISSUE THREAD

**Issue by washimimizuku** [balderdashy/skipper#93]
When you have a massive amount of uploads, having all images on the same folder slows down the machine, it's a common problem. Solving this is simple. Instead of inserting it on one folder:

e.g. uploads/24d5f444-38b4-4dc3-b9c3-74cb7fbbc932.jpg

We can use sub-directories based on the file name:

e.g. uploads/24/d5/24d5f444-38b4-4dc3-b9c3-74cb7fbbc932.jpg

Although we can do that with Skipper by changing the dir ourselves, it would be great to have it as an option on the package.

Do you have something like this planned for a future version? 

Thank you.

**Comment by bozzltron** [comment]
I agree.  I would like to specify the folder I want to upload to.

**Comment by sailsbot** [comment]
Thanks for posting, @washimimizuku.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
